supernovacontroller/sequential/i3c.py

```diff
@@ -185,10 +186,10 @@ def init_bus(self, voltage: int=None, targets=None):
         # TODO: Toggle IBIs off
 
         status = responses[0]["result"]
-        if status == "I3C_BUS_INIT_SUCCESS":
+        if status == "DAA_SUCCESS":
             result = (True, voltage)
         else:
-            result = (False, {"errors": responses[0]["errors"]})
+            result = (False, {"errors": responses[0]["result"]})
```

Comment:
This contains also if comes something in `invalidAddresses`?

Reply:
Is it possible to get invalid addresses via the init bus command?